### ISSUE TITLE
Change tag colour

### ISSUE THREAD

**Issue by phillipgardiner** [open-sdg/open-sdg#1685]
The tag colour currently used for the 'orange/yellow' tags shown below fails accessibility contrast guidelines. An alternative tag colour is therefore required,
![image](https://user-images.githubusercontent.com/60919820/174048446-73486a39-7cf3-422d-b049-9cf1d0b6fdd3.png)

Initial thoughts are, consider using: Hex #707071 (grey), with #FFFFFF writing (white).

@brockfanning - Can I just double check where these tags are used? Are the only used on the goals pages or elsewhere too?

**Comment by phillipgardiner** [comment]
@brockfanning thanks for putting up the PR I'll take a look. Your right the text on the tag does pass, however the colour of the tag against the white background doesn't - this was the primary driver behind the change :)

**Comment by brockfanning** [comment]
Ah, got it, I misunderstood.

**Comment by brockfanning** [comment]
Closing as fixed.
